### PR TITLE
TF-3244 [SEARCH] To chip should include cc, bcc

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#3244

## Resolved

https://github.com/user-attachments/assets/442693a3-a9aa-47c6-bbf1-f97e947e7c8c

### CODE REVIEW

File: test/features/search/search_email_filter_test.dart [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        expect(filter.operator, Operator.OR);
+
+        expect(filter.conditions, {
+          EmailFilterCondition(to: 'singleRecipient@example.com'),
+          EmailFilterCondition(cc: 'singleRecipient@example.com'),
+          EmailFilterCondition(bcc: 'singleRecipient@example.com'),
+        });
+      });
+
+      test('SHOULD generate multiple filters WHEN "to" has multiple items', () {
```
Comment (hoangdat) [inline test/features/search/search_email_filter_test.dart:47]:
```suggestion
      test('SHOULD generate multiple filters which combin with AND operator WHEN "to" has multiple items', () {
```

File: test/features/search/search_email_filter_test.dart [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        // Arrange
+        searchEmailFilter = SearchEmailFilter(
+          to: {'recipient1@example.com', 'recipient2@example.com'},
+        );
+
+        // Act
+        final filters = searchEmailFilter.generateFilterFromToField();
```
Comment (hoangdat) [inline test/features/search/search_email_filter_test.dart:54]: please verify this filter has AND operator

**Review by tddang-linagora — DISMISSED** [review]

File: test/features/search/search_email_filter_test.dart (hunk 2 above)
Comment (dab246) [reply test/features/search/search_email_filter_test.dart:54]: It added to `mappingToEmailFilterCondition` unit test

File: test/features/search/search_email_filter_test.dart (hunk 1 above)
Comment (dab246) [reply test/features/search/search_email_filter_test.dart]: done

**Review by tddang-linagora — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**